### PR TITLE
fix: Make stronger selector for message strip

### PR DESCRIPTION
**Pull request by JKMarkowski**
## Description
There is stronger selector for message-strip close element.

## Screenshots

No visual/usage changes predicted

### CODE REVIEW

**Comment by JKMarkowski** [comment]
Chromatic detected  changes in checkbox/radio buttons. Which i guess are leftovers

File: src/message-strip.scss [hunk 1]
```diff
@@ -67,7 +67,7 @@ $block: #{$fd-namespace}-message-strip;
   min-height: 2rem;
 
   // Elements
-  &__close {
+  &__close.#{$fd-namespace}-button {
```
Comment (jbadan) [inline src/message-strip.scss:70]: I thought we agreed not to do this in the last few meetings?

**Review by jbadan — CHANGES_REQUESTED** [review]
We should not mix `fd-*` classes in the same file. It breaks css modules in fundamental-react.

**Comment by JKMarkowski** [comment]
> We should not mix `fd-*` classes in the same file. It breaks css modules in fundamental-react.

I just added second level of nesting, to make it stronger

**Review by jbadan — APPROVED** [review]

**Review by droshev — APPROVED** [review]

**Pull request merged**